cmd/osctl/cmd/inject.go

```diff
@@ -143,9 +172,13 @@ func newSecurity() *userdata.Security {
 
 func init() {
 	injectCmd.PersistentFlags().StringVar(&crt, "crt", "", "the path to the PKI certificate")
-	helpers.Should(injectCmd.MarkPersistentFlagRequired("crt"))
+	injectOSCmd.MarkFlagRequired("crt")
```

Comment:
`helpers.Should` should be used for all of these since `MarkFlagRequired` returns an `error`.